ts/Maps/Projections/WebMercator.ts

```diff
@@ -0,0 +1,37 @@
+/* *
+ * Web Mercator projection, used for most online map tile services
+ * */
+
+import type { LonLatArray } from '../MapViewOptions';
+import type ProjectionDefinition from '../ProjectionTypes';
+
+'use strict';
```

Comment:
`use strict` should come first, just to be consistent and save time, if someone adds a regular import before it.